sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs

```diff
@@ -5997,7 +5997,7 @@ public partial class KeyValuePairStringObject
     {
         internal KeyValuePairStringObject() { }
         public string Key { get { throw null; } }
-        public object Value { get { throw null; } }
+        public System.BinaryData Value { get { throw null; } }
```

Comment:
This is a key value pair, therefore I believe in most cases, this `Value` will be a primitive type - therefore this should be `object`